docs/reference-guides/core-blocks.md

```diff
@@ -20,7 +20,7 @@ Displays a group of accordion headings and associated expandable content. ([Sour
 
 ## Accordion Heading
 
-Displays an accordion heading. ([Source](https://github.com/WordPress/gutenberg/tree/trunk/packages/block-library/src/accordion-heading))
+Toggles the accordion panel. ([Source](https://github.com/WordPress/gutenberg/tree/trunk/packages/block-library/src/accordion-heading))
```

Comment:
The toggle also displays a heading though, should we have kept a hint for that?

Reply:
I don't have a strong opinion either, but I think it's better if we clearly convey that it also renders the heading.

Reply:
I don't have a strong opinion either, but this block _does_ display a heading, so perhaps it's best to mention it in the description. I'll open a quick PR.